### ISSUE TITLE
mouse key mapping not working with Genshin with play cover 0.7.3

### ISSUE THREAD

**Issue by jimmyszjl** [iVoider/PlayCover#32]
thanks a lot for this project, and I've succeed to install the Genshin (game) with playcover 0.7.3 on my M1 MacBook Pro.
When I tried the key mapping, all looks good, the keyboard mapping works well, but the mouse doesn't. 
1. the left click, middle click and right click can not work although they are mapped.
2. the camera can not move with the mouse move, although the control + M had been mapped. and the camera can only move with the press of the left button of the mouse.

best regards

**Comment by Cyalpaca** [comment]
the same

**Comment by AbelLeo** [comment]
Yep, I also have the problem like that. But the version still works. I wonder if it’s because the Genshin 2.0 resluts it.

**Comment by AlexHow549** [comment]
Exactly! LMB in key mapping not works well. 
app: Genshin Impact 2.1
Device: MacBook Air with Apple M1 chips (Silicon)
